templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+                        Filter by date range to focus on specific timeframes,location to narrow down your search area,
+                        and keywords relevant to the type of imagery or dataset. Additional options, like cloud cover and
+                        sensor type, let you further customize your results to meet quality and relevance standards.
+                    </p>
+                </div>
+                <div id="searchFilterContainer" class="div-search-filter-container">
+                    <div id="mySearchFilters"> </div>
+                    <div id="fullFilterStringContainer" class="div-hidden-filter"></div>
+                    <div id="searchFilterButtonContainer">
+                        {% with %}
+                            {% set button_type = "button_reg_default_alt" %}
+                            {% set button_id = "searchFilterApply" %}
+                            {% set button_label = "Apply" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+
+                        {% with %}
+                                {% set button_type = "button_reg_default_alt" %}
+                                {% set button_id = "searchFilterClear" %}
+                                {% set button_label = "Clear" %}
+                                {% include "partials/button.html" %}
+                            {% endwith %}
+                        </div>
+                </div>
+
+                <div id="searchResultsNavigation" class="search-results-nav">
+                    <div>
+                        {% with %}
+                            {% set button_type = "button_lg_nav_previous" %}
+                            {% set button_id = "searchResultsPrevious" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+
+                        {% with %}
+                            {% set button_type = "button_sm_nav" %}
+                            {% set button_label = "First" %}
+                            {% set button_id = "searchResultsFirst" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+                    </div>
+                    <div>
+                        {% with %}
+                            {% set button_type = "button_lg_nav_next" %}
+                            {% set button_id = "searchResultsNext" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+                    </div>
+
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="container-fluid">
+                    <div id="searchResults"></div>
+                    <div id="datasetDetails">
+                        <div id="datasetContainer" class="div-container-dataset">
+                            <div id="datasetCollectionContainer">
```

Comment:
Text style does not match Figma

![image](https://github.com/user-attachments/assets/b012c90e-e3af-460c-b7ae-adeef5ec859a)